public/css/sections/api.styl

```diff
@@ -142,7 +142,7 @@
         position: absolute
         top: 98%
         right: 0
-        height: 200px
+        // height: 200px
```

Comment:
Why not just remove the line?

Reply:
I thought of in case we would want to revert back at some point hence we'd still have the information of length back then that's all. I can definitely remove it.

Reply:
i would just remove it :P

Reply:
Drop it. Keep it clean.

Reply:
Done